src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comments';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => <CommentInfo comment={comment} />)}
+  </div>
```

Comment:
Next time pls use semantic tags for list rendering like `<ul>` and `<li>,` also you can use this [video ](https://mate.academy/en/learn/react/rendering-lists?section=video&videoId=809)to split list into components